app/upw/templates/placement-details/availability.njk

```diff
@@ -56,7 +56,7 @@
       <form method="post" action="{{ action }}">
         <input type="hidden" name="x-csrf-token" value="{{ csrfToken }}">
 
-        <fieldset class="govuk-fieldset" aria-describedby="individual_availability-legend">
+        <fieldset class="govuk-fieldset scrollable" aria-describedby="individual_availability-legend">
```

Comment:
Allow for horizontal scroll when the element overflows it's container, this shouldn't happen with the way I've configured things but will handle any weird edge cases